features/samples.feature

```diff
@@ -61,7 +61,15 @@ Feature: Basic smell detection
     Then the exit status indicates smells
     And it reports:
     """
-    optparse.rb -- 111 warnings:
+    optparse.rb -- 128 warnings:
+      OptionParser declares the attribute banner (Attribute)
```

Comment:
The description is a little odd if we only flag writers as smelly. How about changing the message, e.g., to `declares the writeable attribute`.